setup.cfg

```diff
@@ -16,19 +16,20 @@ classifiers =
     Intended Audience :: Science/Research
     License :: OSI Approved :: BSD License
     Programming Language :: Python
-    Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
 
 [options]
 zip_safe = False
 include_package_data = True
 packages = find:
-python_requires = >=3.6
+python_requires = >=3.7
 install_requires =
     jinja2
     tornado>=6.1.0
+    tornado_openapi3
```

Comment:
this is rad, didn't know about it. however, if this is an optional feature that won't be used all the time, I'd like to see it as an `extra`, and have at least a bottom pin.